homeassistant/components/neato/strings.json

```diff
@@ -1,26 +1,20 @@
 {
   "config": {
     "step": {
-      "user": {
-        "title": "Neato Account Info",
-        "data": {
-          "username": "[%key:common::config_flow::data::username%]",
-          "password": "[%key:common::config_flow::data::password%]",
-          "vendor": "Vendor"
-        },
-        "description": "See [Neato documentation]({docs_url})."
+      "pick_implementation": {
+        "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
       }
     },
-    "error": {
-      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
-      "unknown": "[%key:common::config_flow::error::unknown%]"
-    },
-    "create_entry": {
-      "default": "See [Neato documentation]({docs_url})."
-    },
     "abort": {
+      "missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]",
+      "authorize_url_timeout": "[%key:common::config_flow::abort::oauth2_authorize_url_timeout%]",
+      "no_url_available": "[%key:common::config_flow::abort::oauth2_no_url_available%]",
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
-      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]"
+      "entry_updated": "Successfully updated"
```

Comment:
Please use the common keys and references as much as possible.

https://github.com/home-assistant/core/blob/d4ab7880af85fa746bfd27754f3fee5c77460734/homeassistant/strings.json#L73

Reply:
Done